packages/curator/spec

```diff
@@ -23,4 +23,5 @@ files:
   # release includes these in the python package. 
   - curator/vendor/certifi-2019.3.9-py2.py3-none-any.whl
   - curator/vendor/requests-2.21.0-py2.py3-none-any.whl
```

Comment:
Does this need to be removed?

Reply:
They are in the upstream python package. But we vendor python. Therefore, we have to copy them to the curator package. So no, it needs to be there. I added the comments so future cg-ers don't remove them.

Reply:
👍